data/tag-categories.json

```diff
@@ -45,30 +49,30 @@
         "order": 2,
         "simple": false,
         "tags": [
-            "arch:esrgan",
+            "arch:cain",
```

Comment:
Why did you sort the arch tags?

Reply:
Sorry, must have been a misclick ☹️

I can revert this

Reply:
Should be resolved now